linkis-computation-governance/linkis-manager/linkis-manager-common/src/main/java/org/apache/linkis/manager/common/entity/persistence/ECResourceInfoRecord.java

```diff
@@ -87,6 +89,14 @@ public String getLabelValue() {
     return labelValue;
   }
 
+  public String getEngineType() {
+    if (StringUtils.isNotBlank(labelValue)) {
+      return labelValue.split(",")[1].split("-")[0];
```

Comment:
may add  annotation for labelValue eg "hadoop-IDE,spark-2.4.3 "

Reply:
ok